src/Component/Authentication/SignUp.jsx

```diff
@@ -0,0 +1,675 @@
+
+                "& .MuiInputBase-root": {
+                  "&:before": {
+                    borderBottom: "none",
+                  },
+                  "&:hover:before": {
+                    borderBottom: "none",
+                  },
+                  "&:after": {
+                    borderBottom: "none",
+                  },
+                },
+              }}
+              error={passwordError}
+              label="Password"
+              variant="filled"
+              type={showPassword ? "text" : "password"}
+              value={passwordInput}
+              onChange={(event) => setPasswordInput(event.target.value)}
+              onBlur={handlePassword}
+              fullWidth
+              size="small"
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <IconButton
+                      onClick={handleClickShowPassword}
+                      onMouseDown={handleMouseDownPassword}
+                    >
+                      {showPassword ? <VisibilityOff /> : <Visibility />}
+                    </IconButton>
+                  </InputAdornment>
+                ),
+              }}
+            />
+
+            <TextField
+              sx={{
+                borderColor: "white",
+                backgroundColor: "white",
+                borderRadius: "1rem",
+                "& .MuiInputBase-root": {
+                  "&:before": {
+                    borderBottom: "none",
+                  },
+                  "&:hover:before": {
+                    borderBottom: "none",
+                  },
+                  "&:after": {
+                    borderBottom: "none",
+                  },
+                },
+              }}
+              id="city"
+              error={cityError}
+              label="City"
+              value={cityInput}
+              onChange={(event) => setCityInput(event.target.value)}
+              onBlur={handleCity}
+              variant="filled"
+              fullWidth
+              size="small"
+            />
+            <FormControl
+              fullWidth
+              size="small"
+              variant="filled"
+              style={{
+                backgroundColor: "white",
+                borderRadius: "1rem",
+                "& .MuiInputBase-root": {
+                  "&:before": {
+                    borderBottom: "none",
+                  },
+                  "&:hover:before": {
+                    borderBottom: "none",
+                  },
+                  "&:after": {
+                    borderBottom: "none",
+                  },
+                },
+              }}
+            >
+              <InputLabel id="state-select-label">State</InputLabel>
+              <Select
+                labelId="state-select-label"
+                value={selectedState}
+                onChange={handleState}
+                label="State"
```

Comment:
When the State Selector is focused, it displays an Underline. Adding **disableUnderline** prop can help to eliminate that.

Reply:
Thanks Nihal, used disableUnderline to remove it.